### PR TITLE
Fixing plugin build issues

### PR DESCRIPTION
**Pull request by fr0hsty**
Plugin now successfully builds for win64, but will not successfully launch until udsdk.dll is added to the resulting plugins build folder.

### CODE REVIEW

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #include "UdsSubpassComposite.h"
 #include "UDSubsystem.h"
-#include "EdGraphSchema_K2_Actions.h"
+// #include "EdGraphSchema_K2_Actions.h"
```
Comment (OsiJr) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.cpp]: Just delete the include

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.h [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #include "UdsSubpass.h"
 
+
```
Comment (OsiJr) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.h]: Extra line?

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDComponent.h [hunk 3]
```diff
@@ -2,41 +2,42 @@
...
+#include "Components/PrimitiveComponent.h"
 #include "UDComponent.generated.h"
 
 UCLASS(Blueprintable, BlueprintType, ClassGroup = (Custom), meta = (BlueprintSpawnableComponent))
 class UUDComponent : public UPrimitiveComponent
 {
 	GENERATED_BODY()
 
-	friend class FPointCloudSceneProxy;
+		friend class FPointCloudSceneProxy;
```
Comment (OsiJr) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDComponent.h]: Weird indentation? And lots below as well.

**Review by OsiJr — CHANGES_REQUESTED** [review]

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.cpp (hunk 1 above)
Comment (samsinsane) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.cpp]: This line can probably just be removed if it's not needed?

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.h (hunk 2 above)
Comment (samsinsane) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Private/Subpasses/UdsSubpassComposite.h]: This isn't necessary.

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDComponent.h (hunk 3 above)
Comment (samsinsane) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDComponent.h]: Not sure why things are indented in this file?

File: Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDDefine.h [hunk 4]
```diff
@@ -5,7 +5,9 @@
 #include "udError.h"
 #include "udPointCloud.h"
 
+
```
Comment (samsinsane) [inline Plugins/UnlimitedDetail/Source/UnlimitedDetail/Public/UDDefine.h]: Not necessary.

**Review by samsinsane — CHANGES_REQUESTED** [review]

**Review by samsinsane — APPROVED** [review]

**Review by OsiJr — APPROVED** [review]

**Pull request merged**